frontend/packages/ux-editor/src/components/config/editModal/EditBooleanValue.tsx

```diff
@@ -44,7 +44,7 @@ export const EditBooleanValue = ({
       helpText={
         isValueExpression(component[propertyKey])
           ? t('ux_editor.component_properties.config_is_expression_message')
-          : helpText
+          : helpText || componentPropertyHelpText(propertyKey)
```

Comment:
I have kept `helpText` here, because in some edge cases, the component can have a prop `hasCustomFileEndings` (if it is still in use?). And then it use the description prop. See `FormComponentConfig.tsx`:
![image](https://github.com/user-attachments/assets/a0950c37-9665-4e49-9974-8851d7778121)

Reply:
It looks like it also shows technical information, so I think we can remove it too:

<img width="759" alt="hascustomfileendings" src="https://github.com/user-attachments/assets/d0f202d1-9d31-4334-be19-1dfa8526e4e3">